modules/audio-form-edit.twig

```diff
@@ -31,43 +31,54 @@
 {% block formHtml %}
     <div class="row">
         <div class="col-md-12">
+            <ul class="nav nav-tabs" role="tablist">
+                <li class="active"><a href="#generalTab" role="tab" data-toggle="tab"><span>{% trans "General" %}</span></a></li>
+                <li><a href="#actionTab" role="tab" data-toggle="tab" class="layout-designer-only"><span>{% trans "Action" %}</span></a></li>
```

Comment:
I don't think we want to add these to each Widget form manually if we can avoid it - there will be loads of custom widgets, etc which will need upgrading as a result.

I imagined we would dynamically add the tab to each form using JavaScript - shall we have a chat about this?